mysql/sql/00_schema_mysql.sql

```diff
@@ -95,29 +96,33 @@ CREATE TABLE `class` (
 --
 
 CREATE TABLE `slot` (
-  `id` INTEGER PRIMARY KEY,
+  `id` INTEGER AUTO_INCREMENT PRIMARY KEY,
   `lesson_type` varchar(3) NOT NULL,
   `day` tinyint(3) NOT NULL,
   `start_time` decimal(3,1) NOT NULL,
   `duration` decimal(3,1) NOT NULL,
   `location` varchar(31) NOT NULL,
   `is_composed` boolean NOT NULL,
   `professor_id` int (11),
-  `last_updated` datetime NOT NULL
+  `class_id` int(11) NOT NULL,
```

Comment:
There should not any `class_id`. This was already removed in another inside the `develop` branch, was this a merge conflict gone wrong?